.markdownlint.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
VS Code looks for it in the root.
Of course, I can live without VS Code highlighting rule violations - if this file is messy here.